integration/containerd/cri/integration-tests.sh

```diff
@@ -287,6 +347,9 @@ main() {
 			make -e test-integration
 	done
 
+	TestContainerMemoryUpdate 1
+	TestContainerMemoryUpdate
```

Comment:
Nit: for consistency, I would have used 1 and 0.

Reply:
Updated.